README.md

```diff
@@ -65,3 +65,26 @@ always define it in the `_variables.scss` file.
 ## Hosting
 The content of this repository is hosted on https://comixcraft.com/ over [Netlify](https://www.netlify.com/).
 Updates on the website are done automatically on update of the `main`-branch.
+
+## Updating the catalog
+The catalog-structure is defined in `server/assets/comixplain-catalog.js`. 
+This catalog follows the same structure as in the [Comixplain repository](https://github.com/fhstp/comixplain/blob/main/docs/assets.json).
+The structure in the `comixplain-catalog.js` is used to generate the catalog on the website. 
+The naming of categories, sub-categories and filters all originate from this catalog, 
+therefore it is important that all keys carry readable names.
+
+### Updating the catalog with a new version from the Comixplain repository
+This paragraph describes how the existing catalog structure from the Comixplain repository most be adjusted to 
+properly work for comixcraft.
+
+1. Add the JSON-structure from the Comixplain repository (`/docs/assets.json`) to be 
```

Comment:
I dont fully understand the first step, do you mean copy the json structure from the repository to then import it into comixplain-catalog.js?

Reply:
Yes

Reply:
the clairification in the readme is great, thanks!